jvm/scalactic-test/src/test/scala/org/scalactic/source/ObjectMetaSpec.scala

```diff
@@ -19,21 +19,23 @@ import org.scalatest._
 
 class ObjectMetaSpec extends funspec.AnyFunSpec with matchers.should.Matchers {
 
-  case class Person(name: String, age: Int) {
+  case class Person(name: String, private val age: Int) {
     val otherField = "test other field"
+    val privateField = "test private field"
```

Comment:
@phucnh  Er, do you actually intended to use a private val here?

Reply:
@cheeseng Thank you for reviewing.
I have fixed https://github.com/scalatest/scalatest/pull/1834/commits/bb6a1e0bdd3a8b18e8cbb3efbc263d597ad785fe